### PR TITLE
doc: add more noob fdly code

### PR DESCRIPTION
**Pull request by AlphaWong**
# background
the quickstart for the javascript is oversimple. it takes time for noob like my self to dig into the source to check the rightway.

# file needs to be changed
https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk/examples/javascript/test/getting_started/firstScript.spec.js

# propose
```js
// doc: https://www.npmjs.com/package/selenium-webdriver?activeTab=code
const {Builder, Browser} = require('selenium-webdriver');

(async function helloSelenium() {
  let driver = await new Builder().forBrowser(Browser.CHROME).build();

  await driver.get('https://selenium.dev');

  await driver.quit();
})();
```
I would create a PR for update the doc for noob to pickup the framework more smoothly

### CODE REVIEW

**Comment by diemol** [comment]
How is `Browser.CHROME` making it more simple?

File: examples/javascript/test/getting_started/firstScript.spec.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
-const {By, Builder} = require('selenium-webdriver');
+// doc: https://www.npmjs.com/package/selenium-webdriver?activeTab=code
```
Comment (diemol) [inline examples/javascript/test/getting_started/firstScript.spec.js]: If you add one more line, the references get broken, why is this line needed?

**Comment by AlphaWong** [comment]
> How is `Browser.CHROME` making it more simple?

as I need to check this file to find the source of Enum
https://github.com/SeleniumHQ/selenium/blob/trunk/javascript/node/selenium-webdriver/lib/capabilities.js#L31

if I know there is a Emun list on the quickstart page, a noob like myself can easily know the expected input rather than looking at the source and getting confused for the README at https://github.com/SeleniumHQ/selenium/tree/trunk/javascript/node/selenium-webdriver#usage

```js
/**
 * Recognized browser names.
 * @enum {string}
 */
const Browser = {
  CHROME: 'chrome',
  EDGE: 'MicrosoftEdge',
  FIREFOX: 'firefox',
  INTERNET_EXPLORER: 'internet explorer',
  SAFARI: 'safari',
}
```

File: examples/javascript/test/getting_started/firstScript.spec.js (hunk 1 above)
Comment (AlphaWong) [reply examples/javascript/test/getting_started/firstScript.spec.js]: good point, i should remove this line

Comment (AlphaWong) [reply examples/javascript/test/getting_started/firstScript.spec.js]: deleted! thx for review!

**Review by diemol — APPROVED** [review]
Thank you, @AlphaWong!
